### ISSUE TITLE
Use Unitful.jl NoUnits as `units` default?

### ISSUE THREAD

**Issue by rafaqz** [rafaqz/FieldMetadata.jl#13]
I've wanted this for ages. It should absolutely be the default instead of `1`, which is only there to let us multiply the field as you do with unitful units.

~~We don't need to depend on Unitful.jl as the function is only run in local scope where Unitful should be loaded. But it's 
a little unusual to do this.~~

Actually we would need to use Requires.jl for this. Hmm.

Do any other users have thoughts on this? @briochemc ?

**Comment by briochemc** [comment]
No, sorry, not sure that I help... Would depending on Unitful for `using Unitful: NoUnits` be so bad though? I mean, I get that you want to keep the dependencies to a minimum, but I think that most people using FieldMetadata also use Unitful anyway, no?
